.github/workflows/Operator-CI.yaml

```diff
@@ -70,20 +70,18 @@ jobs:
           # Add Prometheus ServiceMonitor CRD to avoid BridgeIngress Condition Ready: False Type: PrometheusUnavailable
           kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/v0.9.0/manifests/setup/prometheus-operator-0servicemonitorCustomResourceDefinition.yaml
       - name: Deploy Shard operator
+        env:
+          DEV_EVENT_BRIDGE_SSO_CLIENT_ID: ${{ secrets.DEV_EVENT_BRIDGE_SSO_CLIENT_ID }}
+          DEV_EVENT_BRIDGE_SSO_SECRET: ${{ secrets.DEV_EVENT_BRIDGE_SSO_SECRET }}
         run: |
           kubectl create namespace $NAMESPACE
-          # Operator requires Keycloak instance to be available, install it
-          kubectl create secret generic keycloak-realm --from-file=event-bridge-fm-realm-sample.json=kustomize/overlays/minikube/keycloak/event-bridge-fm-realm-sample.json -n $NAMESPACE
-          kubectl create secret generic keycloak-secrets --from-literal=KEYCLOAK_USER=admin --from-literal=KEYCLOAK_PASSWORD=123 -n $NAMESPACE
-          # Not ideal as kustomize is used for different purposes but it is the easiest approach without duplications
-          kubectl create -f kustomize/overlays/minikube/keycloak/keycloak.yaml -n $NAMESPACE
-          kubectl wait --for=condition=available --timeout=120s deployment/keycloak -n $NAMESPACE
           # Deploy operator
           kubectl apply -f shard-operator/target/kubernetes/bridgeexecutors.com.redhat.service.bridge-v1.yml
           kubectl apply -f shard-operator/target/kubernetes/bridgeingresses.com.redhat.service.bridge-v1.yml
           kubectl apply -f shard-operator/target/kubernetes/minikube.yml
-          # Point SSO URL to the Keycloak Service port, use HTTP to avoid complications with certificate
-          kubectl set env deployment/shard-operator EVENT_BRIDGE_SSO_URL=http://keycloak:8180/auth/realms/event-bridge-fm -n $NAMESPACE
+          kubectl set env deployment/shard-operator EVENT_BRIDGE_SSO_URL=https://sso.redhat.com/auth/realms/redhat-external -n $NAMESPACE
```

Comment:
@sutaakar Similar question here: It looks like these values are fixed, so can they be included directly in the `overlay` for our CI pipeline rather than us having to run `set env` commands?

Reply:
In this case I am using yaml files generated by Quarkus and not Kustomize.
I need to try if it is possible to replace it with direct Kustomize invocation

Reply:
@sutaakar My comment here is definitely not a blocker for this PR as the process already exists. It was more a question around something we could improve in the future if it was possible.

cheers,
Rob